components/vf-design-tokens/src/theme.palette.alias.yml

```diff
@@ -0,0 +1,53 @@
+aliases:
+
+  # EMBL Brand Colours
+  # ------------------
+
+  vf-color--brand: '#18974C'
+  vf-color--brand--dark: '#0A5032'
+
+
+  # Text Colours
+  # ------------
+
+  vf-color__text--primary: '#000000'
+  vf-color__text--secondary: '#373A36'
+
+  # Interactive Text Colours
+
+  vf-color__link: '#3B6FB6'
+  vf-color__link--hover: '#193f90'
+  vf-color__link--focus: '#193f90'
+  vf-color__link--visited: '#563D82'
+
+  # Background Colours
+  # ------------------
+
+  vf-color__background--primary: '#FFFFFF'
+  vf-color__background--secondary: '#D0D0CE'
+  vf-color__background--tertiary: '#54585A'
+
+
+  # Button Colours
+  # --------------
+
+  # Primary Button
+
+  vf-color__button__text--primary: '#FFFFFF'
+  vf-color__button__border--primary: '#3B6FB6'
+  vf-color__button__background--primary: '#3B6FB6'
+  vf-color__button__shadow--primary: '#193f90'
+
+  # Secondary Button
+
+  vf-color__button__text--secondary: '#3B6FB6'
```

Comment:
As this is "EMBL Blue", is there a way to avoid repeating the hex value ... or I suppose the (evenutal) Figma token export should take care of that extrapolation?

Reply:
Yeah, it _should_ do I think.

The only way to stop the duplication in code is to actually move everything to Style Dictionary - as Theo cannot do this.

It's a move we probably need to make at some point soon anyhow.